contrib/runners/cloudslang/runner.yaml

```diff
@@ -0,0 +1,16 @@
+- aliases: []
+  description: A runner to execute cloudslang flows.
+  enabled: true
+  name: cloudslang
```

Comment:
IMO this PR is big enough. I agree this should/could be dropped, but should be done in a different PR.